zap/src/main/java/org/zaproxy/zap/view/MultipleRegexesOptionsPanel.java

```diff
@@ -76,6 +82,67 @@ public MultipleRegexesOptionsPanel(Dialog owner, RegexesTableModel model) {
 
         this.owner = owner;
         getTable().setSortOrder(0, SortOrder.ASCENDING);
+        JButton importButton =
+                new JButton(Constant.messages.getString("context.include.import.label"));
+        importButton.addActionListener(e -> importButtonClicked());
+        addButton(importButton);
+    }
+
+    /**
+     * Checks if {@code regex} is contained in {@code regexes} and if it is a valid regex pattern.
+     * If {@code regexes} is null, then the duplication check is omitted
+     *
+     * @param regex A regex as string
+     * @param regexes List of regex strings
+     * @return True if {@code regex} is not a duplicate and the pattern is valid.
+     * @throws PatternSyntaxException If {@code regex} is not a valid regex pattern
+     */
+    private static boolean validateRegexAndCheckForDuplicates(
+            String trimmedRegex, List<String> regexes) throws PatternSyntaxException {
+        if (trimmedRegex == null
+                || trimmedRegex.isEmpty()
+                || regexes != null && regexes.contains(trimmedRegex)) {
+            return false;
+        }
+        Pattern.compile(trimmedRegex, Pattern.CASE_INSENSITIVE);
+        return true;
+    }
+
+    private void importButtonClicked() {
+        File file = selectImportFile();
+        if (file != null) {
+            List<String> regexes = new ArrayList<String>(getRegexes());
+
+            try (BufferedReader br = new BufferedReader(new FileReader(file))) {
+                String line = null;
+                while ((line = br.readLine()) != null) {
+                    String trimmedLine = line.trim();
+                    if (MultipleRegexesOptionsPanel.validateRegexAndCheckForDuplicates(
```

Comment:
- The call does not need to be qualified with the class name.
 - It would be better to add the braces to the if statement, for consistency.

Reply:
Isn't it best practice to call static methods in a static way, i.e. by calling it via the class name?
Added braces to if

Reply:
I'm not aware of a best practice that requires qualifying the static method calls in the same class (they are called statically regardless).

Reply:
You are right, in the same class it is not required, Changed it.

Reply:
The call in `validateFields` can be changed as well, same reason.

Reply:
done